apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/test/resources/expected/expected-mcs.ndjson

```diff
@@ -1,4 +1,4 @@
+{"timestamp":"2022-01-25T15:02:35Z","seq":"2","changeType":"CHANGE_TYPE_UPDATE","claim":{"idrClmHdIcn":"-5389afa7276739","idrContrId":"00000","idrClaimTypeEnum":"CLAIM_TYPE_MEDICAL","idrBeneLast16":"Arella","idrBeneFirstInit":"R","idrBeneSexEnum":"BENEFICIARY_SEX_MALE","idrStatusCodeEnum":"STATUS_CODE_ACTIVE_A","idrStatusDate":"1970-01-01","idrBillProvEin":"XX-XXXXXXX","idrBillProvType":"20","idrBillProvSpec":"01","idrTotBilledAmt":"1571.11","idrClaimReceiptDate":"1970-01-01","idrClaimMbi":"1S00E00AA01","mcsDiagnosisCodes":[{"idrClmHdIcn":"-5389afa7276739","idrDiagIcdTypeEnum":"DIAGNOSIS_ICD_TYPE_ICD10","idrDiagCode":"J329"}],"mcsDetails":[{"idrDtlFromDate":"1976-03-31","idrDtlToDate":"1976-03-31"}],"idrHdrToDos":"1976-03-31"}}
+{"timestamp":"2022-01-25T15:02:35Z","seq":"3","changeType":"CHANGE_TYPE_UPDATE","claim":{"idrClmHdIcn":"-da6f0db3a56d40","idrContrId":"00000","idrClaimTypeEnum":"CLAIM_TYPE_MEDICAL","idrBeneLast16":"Arella","idrBeneFirstInit":"R","idrBeneSexEnum":"BENEFICIARY_SEX_MALE","idrStatusCodeEnum":"STATUS_CODE_ACTIVE_A","idrStatusDate":"1970-01-01","idrBillProvEin":"XX-XXXXXXX","idrBillProvType":"20","idrBillProvSpec":"01","idrTotBilledAmt":"889.66","idrClaimReceiptDate":"1970-01-01","idrClaimMbi":"1S00E00AA01","mcsDiagnosisCodes":[{"idrClmHdIcn":"-da6f0db3a56d40","idrDiagIcdTypeEnum":"DIAGNOSIS_ICD_TYPE_ICD10","idrDiagCode":"J329"}],"mcsDetails":[{"idrDtlFromDate":"1979-04-04","idrDtlToDate":"1979-04-04"}],"idrHdrToDos":"1979-04-04"}}
+{"timestamp":"2022-01-25T15:02:35Z","seq":"4","changeType":"CHANGE_TYPE_UPDATE","claim":{"idrClmHdIcn":"-6ec0275272718c","idrContrId":"00000","idrClaimTypeEnum":"CLAIM_TYPE_MEDICAL","idrBeneLast16":"Arella","idrBeneFirstInit":"R","idrBeneSexEnum":"BENEFICIARY_SEX_MALE","idrStatusCodeEnum":"STATUS_CODE_ACTIVE_A","idrStatusDate":"1970-01-01","idrBillProvEin":"XX-XXXXXXX","idrBillProvType":"20","idrBillProvSpec":"01","idrTotBilledAmt":"1656.54","idrClaimReceiptDate":"1970-01-01","idrClaimMbi":"1S00E00AA01","mcsDiagnosisCodes":[{"idrClmHdIcn":"-6ec0275272718c","idrDiagIcdTypeEnum":"DIAGNOSIS_ICD_TYPE_ICD9","idrDiagCode":"J329"},{"idrClmHdIcn":"-6ec0275272718c","idrDiagIcdTypeEnum":"DIAGNOSIS_ICD_TYPE_ICD10","idrDiagCode":"E669"}],"mcsDetails":[{"idrDtlFromDate":"1982-01-20","idrDtlToDate":"1982-01-20"}],"idrHdrToDos":"1982-01-20"}}
```

Comment:
Are we still properly covered on ICD9 logic since this switches a few for ICD10?

Reply:
There's an ICD9 entry in the line 4 of the code you referenced, covering that code path.

Reply:
I saw that one, just wanted to make sure the ones changed werent covering something we needed.